style.css

```diff
@@ -486,12 +582,28 @@ header {
     margin-left: 0;
     margin-top: 10px;
   }
+
+  #contact-container {
+    padding: 12% 30%;
+    background-image: url(./asset/contact-form-bg-desktop.svg);
+    background-size: 94%;
+    background-position: 60% 30%;
+  }
```

Comment:
- [ ] All the elements in the contact section are centered on the page including the button in the figma design but in your design, the button is not centered on the desktop screen as it should be, so kindly consider fixing it to make it identical to the figma design.
your design
![vc](https://user-images.githubusercontent.com/96683843/230520628-0785e573-ef53-407c-83af-690d1ee48a05.jpg)

figma design
![f](https://user-images.githubusercontent.com/96683843/230520645-224f8fbe-ebc2-4c02-891d-faaeb3824ae5.jpg)